scripts/push-to-assets-branch

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+if [[ -z ${1} || -z ${2} ]]; then
+    echo "Usage: ./scripts/push-to-assets-branch <package-branch> <assets-branch>"
+    exit 1
+fi
+remote=${1}
+branch=${2}
+
+rm -rf /tmp/tmp-assets
+mkdir -p /tmp/tmp-assets
+for i in assets/ index.yaml index.html sha256sum/ charts; do
+    if [[ -d $i ]]; then
+        cp -R $i /tmp/tmp-assets
+    fi
```

Comment:
`index.yaml` and `index.html` will never be copied based on this logic since we only copy directories in.

Also, should we fail out of `push-to-assets-branch` if any of these directories doesn't exist? I'm concerned about a situation where the `charts/` directory fails to get created in an earlier script but the `assets/` succeeds and we end up out of sync.

Reply:
whoops, will fix it. If it doesn't exist that means we are not generating anything and we shouldn't fail. If something failed on the previous step, this script should not be executed.